pkg/kubectl/describe/versioned/describe.go

```diff
@@ -23,7 +23,7 @@ import (
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/kubernetes/pkg/kubectl/describe"
 	"k8s.io/kubernetes/pkg/printers"
-	printersinternal "k8s.io/kubernetes/pkg/printers/internalversion"
+	printersexternal "k8s.io/kubernetes/pkg/printers/externalversion"
```

Comment:
Move this external printers package to `pkg/kubectl/printers`

Reply:
Also, merge this file (`pkg/kubectl/describe/versioned/describe.go`) with `pkg/kubectl/describe/describe.go` and delete this `versioned` directory, we don't need it anymore